cli/tests/integrations/test_package.py

```diff
@@ -671,6 +669,46 @@ def test_search_table():
     assert stderr == b''
 
 
+def test_search_ends_with_wildcard():
```

Comment:
`git add -p`! I like separate commits b/c it makes the git history cleaner. and is easier to revert if necessary. We can decide whether we not to do this or not cli? @jsancio @ssk2 thoughts?